src/api/app/models/package_issue.rb

```diff
@@ -25,7 +25,9 @@ def self.sync_relations(package, issues)
 
         # set change value for all
         issues.each do |pair|
-          PackageIssue.where(package: package, issue: pair.last).lock(true).update_all(change: pair.first)
+          PackageIssue.where(package: package, issue: pair.last).lock(true).find_each do |package_issue|
+            package_issue.update(change: pair.first)
```

Comment:
It is slower yes, but we don't skip validations. If not we can end with corrupted data.